web/src/navbar_alerts.ts

```diff
@@ -11,18 +11,25 @@ import render_insecure_desktop_app_alert_content from "../templates/navbar_alert
 import render_navbar_alert_wrapper from "../templates/navbar_alerts/navbar_alert_wrapper.hbs";
 import render_profile_incomplete_alert_content from "../templates/navbar_alerts/profile_incomplete.hbs";
 import render_server_needs_upgrade_alert_content from "../templates/navbar_alerts/server_needs_upgrade.hbs";
+import render_time_zone_update_offer_content from "../templates/navbar_alerts/time_zone_update_offer.hbs";
 
+import * as browser_history from "./browser_history.ts";
+import * as channel from "./channel.ts";
 import * as desktop_notifications from "./desktop_notifications.ts";
+import * as feedback_widget from "./feedback_widget.ts";
+import {$t} from "./i18n.ts";
 import * as keydown_util from "./keydown_util.ts";
 import type {LocalStorage} from "./localstorage.ts";
 import {localstorage} from "./localstorage.ts";
 import {page_params} from "./page_params.ts";
 import * as people from "./people.ts";
 import {current_user, realm} from "./state_data.ts";
+import * as timerender from "./timerender.ts";
 import {should_display_profile_incomplete_alert} from "./timerender.ts";
 import * as unread from "./unread.ts";
 import * as unread_ops from "./unread_ops.ts";
 import * as unread_ui from "./unread_ui.ts";
+import * as user_settings from "./user_settings.ts";
```

Comment:
This import is typically `import {user_settings} from "./user_settings.ts"` to avoid `user_settings.user_settings.` nonsense.